apps/condo/package.json

```diff
@@ -51,6 +51,7 @@
     "@zeit/next-less": "^1.0.1",
     "ajv": "^8.2.0",
     "ajv-formats": "^2.1.0",
+    "amplitude-js": "^8.18.1",
```

Comment:
MIT

Reply:
@pahaz can't we use packages under MIT license? As far as I know, everything is within the rules described in the [article](https://en.wikipedia.org/wiki/MIT_License)? Or another [[source](https://softwareengineering.stackexchange.com/questions/103273/can-i-use-mit-licence-plugins-in-my-commercial-web-site)]?